android-example/rust/Cargo.toml

```diff
@@ -18,6 +19,7 @@ android_logger = "0.8"
 gl = "0.14"
 jni = "0.13"
 log = "0.4"
+winit = { version = "0.28.1", features = [ "android-game-activity" ] }
```

Comment:
Is there any particular reason for using 'android-game-activity' here? The build failure due to missing arm-clang++ seems to go way when using the 'android-native-activity' feature which doesn't require compilation of C++ files.

Reply:
@mukilan: The reason is that I misread the `winit` docs. I'll switch this over.